packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -845,10 +851,174 @@ export default class Meeting extends StatelessWebexPlugin {
      */
     this.transcription = undefined;
 
+    /**
+     * Password status. If it's PASSWORD_STATUS.REQUIRED then verifyPassword() needs to be called
+     * with the correct password before calling join()
+     * @instance
+     * @type {PASSWORD_STATUS}
+     * @public
+     * @memberof Meeting
+     */
+    this.passwordStatus = PASSWORD_STATUS.UNKNOWN;
+
+    /**
+     * Information about required captcha. If null, then no captcha is required. status. If it's PASSWORD_STATUS.REQUIRED then verifyPassword() needs to be called
+     * with the correct password before calling join()
+     * @instance
+     * @type {Object}
+     * @property {string} captchaId captcha id
+     * @property {string} verificationImageURL Url of the captcha image
+     * @property {string} verificationAudioURL Url of the captcha audio file
+     * @property {string} refreshURL Url used for refreshing the captcha (don't use it directly, call refreshCaptcha() instead)
+     * @public
+     * @memberof Meeting
+     */
+    this.requiredCaptcha = null;
+
+    /**
+     * Indicates the reason for last failure to obtain meeting.meetingInfo. Undefined if meeting info was
+     * retrieved successfully
+     * @instance
+     * @type {MEETING_INFO_FAILURE_REASON}
+     * @public
+     * @memberof Meeting
+     */
+    this.meetingInfoFailureReason = undefined;
```

Comment:
It is required by the web client (we need to show only 1 error - either for password or for captcha). Wbxappapi has some bugs/limitations and having this field makes it easier for the web app to know what caused the failure (captcha or password). Also in cases where something else fails, this field will give you an indication why meetingInfo is not available (although currently I've only put 1 value for that: "other", but it can be expanded). So I've added it thinking that if it's useful for web client then it will probably be useful for others. I can make it private for now and keep using it in the web client or I can remove it completely from the SDK and put that logic in the web client if you prefer.